packages/api/src/hooks/useAllWalletAccounts.ts

```diff
@@ -0,0 +1,72 @@
+import { useMemo } from 'react';
+import useCurrencyConfig from './useCurrencyConfig';
+import useWalletAccountsList from './useWalletAccountsList';
+import useAvailableWallets from './useAvailableWallets';
+
+/** A custom hook that gets the list of added and non-added wallet accounts. */
+const useAllWalletAccounts = () => {
+    const { data: non_added_wallets, ...rest } = useAvailableWallets();
+    const { data: added_wallets } = useWalletAccountsList();
+    const { getConfig } = useCurrencyConfig();
+
+    /** Get all the added and non-added wallets account */
+    const modified_wallets = useMemo(() => {
+        const non_virtual_wallets = added_wallets?.filter(wallet => !wallet.is_virtual);
+
+        /** Modify added wallets list with adding `is_added` and `is_crypto` flag */
+        const modified_added_wallets = non_virtual_wallets?.map(wallet => ({
+            currency: wallet.currency,
+            landing_company_name: wallet.landing_company_name,
+            is_added: true,
+            is_crypto: wallet.currency ? getConfig(wallet.currency)?.is_crypto : false,
```

Comment:
```suggestion
            currency: wallet.currency,
            landing_company_name: wallet.landing_company_name,
            is_added: true,
            is_crypto: wallet.currency ? getConfig(wallet.currency)?.is_crypto : false,
```
This transform is similar both, do for below

Reply:
The data structure for added and non-added wallets are different. I need to transform it here 😢

Reply:
😅  ok its fine then @farhan-nurzi-deriv